org.opent2t.sample.hub.superpopular/com.smartthings.hub/js/thingTranslator.js

```diff
@@ -161,6 +165,24 @@ class Translator {
         });
     }
 
+    _subscribe(deviceId) {
```

Comment:
This doesn't appear to abide by the schema: _subscribe(deviceType, deviceId, callbackUrl, verificationRequest)